src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  justify-content: space-between;
+  width: 100vw;
+  align-items: center;
+}
+
+.logo {
+  cursor: pointer;
+  width: 40px;
+  height: 40px;
+  padding: 10px 0;
+  margin-left: 50px;
+}
+
+.nav__link {
+  display: flex;
+  white-space: nowrap;
+  height: 60px;
+  margin: 0 auto;
+}
+
+.nav__item {
+  display: flex;
+  align-items: center;
+  padding: 10 20px;
+  margin: 0;
+  text-align: center;
+}
+
+.links {
+  display: block;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  text-transform: uppercase;
+  margin: 0 10px;
+  width: auto;
+  align-items: center;
+  text-decoration: none;
+  color: #000;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+  padding-right: 40px;
+}
+
+.links:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  left: 0;
+  bottom: -18.5px;
```

Comment:
```suggestion
  bottom: 0;
```

Should work as well

Reply:
It doesnt work. I this case the 'is-active' underline is right under the Apple